tests/unit/askBirthday.test.ts

```diff
@@ -81,3 +101,35 @@ describe("Member joined channel", () => {
     );
   });
 });
+
+describe("getUsersWhoseBirthdayIsMissing", () => {
+  beforeAll(async () => {
+    await testDb.delete(users);
+  });
+  afterEach(async () => {
+    vi.clearAllMocks();
+    await testDb.delete(users);
+  });
+
+  const mockUser = [
+    { birthday: null, id: constants.userId, teamId: constants.team_id },
+  ];
+
+  it("should return with the mock user", async () => {
+    await testDb.insert(users).values(mockUser);
+    const result = await getUsersWhoseBirthdayIsMissing();
+
+    expect(result).toEqual(mockUser);
+  });
+
+  it("shouldn't find any users", async () => {
+    await testDb
+      .update(users)
+      .set({ birthday: dayjs.utc().toDate() })
+      .where(eq(users.id, mockUser[0].id));
```

Comment:
This test is fake.

You are clearing the db in the aftereach, so this update is not going to find a user, so the result will always be empty.

In general tests should be independent, they should setup separately (i.e create user in db)

Reply:
Nice catch :D